docs/rules/no-typeof-undefined.md

```diff
@@ -9,6 +9,8 @@
 
 Checking if a value is `undefined` by using `typeof value === 'undefined'` is needlessly verbose. It's generally better to compare against `undefined` directly. The only time `typeof` is needed is when a global variable potentially does not exists, in which case, using `globalThis.value === undefined` may be better.
 
+Historical note: Comparing against `undefined` without `typeof` was frowned upon until ES5. This is no longer a problem since all engines currently in use no longer allow reassigning the `undefined` global.
```

Comment:
Is this more correct?

Reply:
Indeed broken on IE8 👍 

<img width="762" alt="Screen Shot 10" src="https://user-images.githubusercontent.com/1402241/204269632-888d305b-aa37-4a24-9264-d68f0b141c06.png">

Reply:
Brokener on IE9 😂 

<img width="683" alt="Screen Shot 11" src="https://user-images.githubusercontent.com/1402241/204269765-f1953347-dd9e-4cb2-9004-dd40a55a17e4.png">